### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Gentilitysp**
In this project I was able to achieve the following:

-  Styling of the desktop version using media queries
-  Updating of my Readme
-  Fixing linters error
-  Creating more background images to suit with desktop version style.
- Making the desktop version responsive
etc

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,698 @@
...
+header div.collapse-menu ul li {
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-weight: 600;
+  line-height: 20px;
+  letter-spacing: 0;
+  cursor: pointer;
+}
+
+@media screen and (min-width: 991.99px) {
```
Comment (safafa) [inline style.css]: - [ ] I think it's better to set  min-width in the range  [992px - 1100px] for a desktop breakpoint.

File: index.html [hunk 2]
```diff
@@ -1,42 +1,314 @@
...
+      </ul>
+    </section>
+
+    <section class="recent-works">
+      <div class="h2-and-line">
+        <h2 class="title fonts-1 fw-n">My Recent Works</h2>
+      </div>
+      <section class="frame44">
+        <ul class="card">
+          <li class="image-box">
```
Comment (safafa) [inline index.html]: image

Your project looks a bit different from the original design.
- [ ]  As you can see in the picture the my works section doesn't take all the page width it's centered and has some margin right and left. 
- [ ] The featured project image takes 60% of the page's width and is aligned to the left, the description takes the remaining 40% and is aligned to the right.
- [ ] For the other projects. The description of each of these projects is on top of their image with a dark opacity with white color text. When hovered on with the mouse the description and dark opacity disappear and a See Project button appears. See the picture bellow
image
Kindly fix these issues. Following the original design is required for this project. As it's an important skill for a front-end dev

File: index.html [hunk 3]
```diff
@@ -1,42 +1,314 @@
...
+
+    <section class="about headline">
+      <div class="about-content">
+        <div class="padding-lf">
+          <h2 class="about-head primary-text-h1 fonts-1">About me</h2>
+          <p class="about-text content p-color fonts-4">Hello I’m a software developer! I can help
+            you build a product , feature or website
+            Look through some of my work and
+            experience! If you like what you see and
+            have a project you need coded, don’t
```
Comment (safafa) [inline index.html]: image

- [ ] Kindly add the missing line in the about me section.
- [ ] The titles language, frameworks and skills should be aligned.
- [ ] Kindly add some margins. Check [the original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) for refrence.

File: index.html [hunk 4]
```diff
@@ -1,42 +1,314 @@
...
+      </ul>
+    </section>
+  <section class="form">
+    <h2 class="inter-font contact-title">
+      I'm always interested in hearing about new projects,
+      so if you'd like to chat please get in touch.
+    </h2>
+    <form action="https://formspree.io/f/mgerpyar" method="post" class="contact-form">
+      <div class="form-control">
+        <label for="name" class="sr-only">Full Name</label>
```
Comment (safafa) [inline index.html]: image

- [ ] As you can see in the picture. The text and the form are aligned. And the text is aligned to the left. Kindly fix these issues.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @Gentilitysp @zieeco , You did a great job so far 👏🏼 👏🏼 

- The project is responsive

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 1 above)
Comment (Gentilitysp) [reply style.css]: Hello,

Thanks for your review. 
I wish to state that on this particular section, if I should declare the range as you stated, the project  wouldn't be responsive is screens with width above 1100px, which means the project is no longer responsive.

Thanks

**Review by AshakaE — APPROVED** [review]
### Hi @Gentilitysp  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**